packages/slate-react/src/hooks/android-input-manager/use-android-input-manager.ts

```diff
@@ -26,6 +26,9 @@ export function useAndroidInputManager({
   node,
   ...options
 }: UseAndroidInputManagerOptions) {
+  /* eslint-disable react-hooks/rules-of-hooks */
```

Comment:
Another option here would be to move the IS_ANDROID check outside of the function, like

```tsx
export const useAndroidInputManager = !IS_ANDROID ? () => null : ({
  node,
  ...options
}: UseAndroidInputManagerOptions) => {
```